Makefile

```diff
@@ -0,0 +1,71 @@
+.PHONY: clean clean-test clean-pyc clean-build
```

Comment:
Please remove Makefile because:
- you can use local aliases for repeated tasks (.bashrc, .zshrc)
- setup commands usually runs only one time

Reply:
The Makefile... Is... is like a correct instruction what are you able to do with the source code. 
All main commands (in my opinion) should be run in ONE way via Makefile for everybody.
If you don't mind let's wait @ivlevdenis comment about it.

Reply:
removing for now, as you requested. Will be able to revert Makefile later

Reply:
I'm use Makefile as one entry point on all of started by me projects. This for simplified enter new developers to project.

Reply:
@ShmakovVA pls, revert back Makeile

Reply:
ok